### DISCUSSION TITLE
Holding cursor and folder over drop zone text causes flash

### DISCUSSION THREAD

**Discussion by littlejustinh** [rpldy/react-uploady#255] [Q&A] [answered]
[Here is a code sandbox](https://codesandbox.io/s/crop-and-upload-react-uploady-fixed-aspect-simple-forked-y9xh3?file=/src/App.js)

I'm sure i've done something wrong with the style, but i believe i pulled it from the examples.

When I drag a folder over the dropzone, the drop zone goes crazy if you hover over top of the text before dropping the file. Most of the time this also results in an unsuccessful drop. 

What am i doing wrong?

**Comment by yoavniran** [comment, marked as answer]
Hey @littlejustinh
Actually this is a problem in the `upload-drop-zone` component. You're not doing anything wrong :)
Im working on a fix now and will have it out shortly
thanks for reporting!

**Reply by yoavniran** [reply]
@littlejustinh 0.14.2 released
please confirm it fixes the problem for you

**Comment by littlejustinh** [comment]
Awesome! Thanks for fixing that so quickly, faster than I could even reply to your first comment. Everything looks good thank you so much!
